index.html

```diff
@@ -0,0 +1,93 @@
+<!Doctype HTML>
```

Comment:
There's so much going on in your HTML code. This is why you have modules.

See if you can reduce this to very few lines.

Reply:
Okay